### DISCUSSION TITLE
Fix MC-210802

### DISCUSSION THREAD

**Discussion by emielderckx** [PurpurMC/Purpur#451] [__Completed__] [answered]
I don't know how much Purpur can do about something like this but talking about it can never hurt. It would be cool if Purpur could have some sort of config option to fix this bug.

https://bugs.mojang.com/browse/MC-210802

**Comment by BillyGalbreath** [comment]
Is this still a thing now that 1.18 uses a circular view distance?

**Reply by emielderckx** [reply]
Yep, it's still there (doesn't involve a player) and the spawn chunks haven't been changed 

(Tested it on the latest Purpur build 1496)

**Comment by BillyGalbreath** [comment, marked as answer]
Ok, after countless hours of researching this bug, I managed to find exactly what makes the grass grow and applied the same logic on the eat block pathfinder.

Implemented in build 1506

https://github.com/PurpurMC/Purpur/commit/84028e28a579bd38b52ef10a2ebb6520b55f36fc
